pkg/subctl/cmd/deploybroker.go

```diff
@@ -203,7 +203,9 @@ func isValidGlobalnetConfig() (bool, error) {
 	if err != nil || defaultGlobalnetClusterSize == 0 {
```

Comment:
@vthapar In the API CreateGlobalnetConfigMap() of this file, we will check if there is already an existing GlobalnetConfigMap on the Broker and avoid creating it if present. Shall we check the default globalCIDR that is present in the config Map and update it to the new CIDR (242.0.0.0/8). Although we are not handling upgrades, this is an easy step and will go a long way to help our users who forget to delete the globalnetConfigMap on the Broker. What do you say?

CC: @mangelajo @vthapar @tpantelis @nyechiel

Reply:
If we don't want to support upgrades, we shouldn't support this. Detecting old CIDR and changing it to new falls under supporting upgrades. It will cause confusion if some aspects of an upgrade are handled and others not. As long as we throw error if old one is using an unsupported type, it should be fine.

Reply:
In that case, we should return error when deploy-broker is executed and Broker already has a globalnetConfigMap with old CIDR. Can you please include the change as part of CreateGlobalnetConfigMap handling.

Reply:
Couldn't add it to CreateGlobalnetConfigMap() coz it is in broker and would've resulted in a cyclic dependency between globalnet and broker. Maybe move CreateGlobalnetConfigMap to globalnet instead?